app/src/main/java/com/kms/cura/view/fragment/RegDoc_AccountFragment.java

```diff
@@ -192,4 +247,57 @@ private boolean setErrorMessageRePwd() {
         }
         return true;
     }
+
+    private List<SpecialityEntity> getSpecialitySelected(){
+        boolean[] specialities = bundle.getBooleanArray(RegisterDoctorActivity.SPECIALITY);
+        List<SpecialityEntity> specialitySelected = new ArrayList<>();
+        List<SpecialityEntity> allSpecialities = SpecialityModel.getInstace().getSpecialities();
```

Comment:
Call this through SpecialityController, dont't call straight into SpecialityModel like this